### PR TITLE
Styling fixes

### PR DESCRIPTION
**Pull request by jsflca2012**
addresses #214 and #228

### CODE REVIEW

**Comment by NeilAllavarpu** [comment]
Why were new files created instead of editing the pre-existing files?

**Comment by jsflca2012** [comment]
> Why were new files created instead of editing the pre-existing files?

What do you mean by this? I did not intend to create any new files.

File: src/components/FeatureViews/Accounts/ProfileCard.js [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+import {Card, Paper, Typography} from "@material-ui/core";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import ListView from "@material-ui/icons/ViewList";
+import CardView from "@material-ui/icons/ViewModule";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/ProfileCard.js]: Are these extra imports necessary? If not, remove them.

File: src/components/FeatureViews/Search/cards/AccountsCards.js [hunk 2]
```diff
@@ -53,16 +55,23 @@ function AccountsCards(props) {
 
     });
 
-    const fullName = props.user.user ? `${props.user.user.first_name} ${props.user.user.last_name}`: "a";
+    // console.log(props.user);
+
+    const fullName = `${props.user.user.first_name} ${props.user.user.last_name}`;
+
+    useEffect(()=>{
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/cards/AccountsCards.js]: Remove empty `useEffect`

File: src/components/FeatureViews/Registration/SelectParentDialog.js [hunk 3]
```diff
@@ -3,110 +3,160 @@ import PropTypes from 'prop-types';
...
-                        last_name: selectedParent.user.last_name,
-                        name: selectedParent.user.first_name + " " + selectedParent.user.last_name,
-                    },
-                    user_uuid: selectedParent.user.id,
-                    gender: selectedParent.gender,
-                    birth_date: selectedParent.birth_day,
-                    student_list: selectedParent.student_list,
-                    account_type: "PARENT",
-                };
+            console.log(selectedParent);
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Registration/SelectParentDialog.js]: Remove console.log

File: src/components/FeatureViews/Registration/SelectParentDialog.js [hunk 4]
```diff
@@ -3,110 +3,160 @@ import PropTypes from 'prop-types';
...
+        first_name: "Eileen",
+        last_name: "Hong",
+        name: "Eileen Hong",
+    },
+    user_uuid: 12,
+    gender: "F",
+    birth_date: "12/12/1963",
+    student_list: [1, 2],
+    account_type: "PARENT",
+};
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Registration/SelectParentDialog.js]: Remove these

File: src/components/FeatureViews/Registration/SelectParentDialog.js [hunk 5]
```diff
@@ -3,110 +3,160 @@ import PropTypes from 'prop-types';
...
-                    justify="center"
-                >
-                    <Grid item>
-                        <AccountsCards user={this.props.registration.CurrentParent}/>
-                    </Grid>
-                </Grid>
+                <div className="cardsPadding">
+                    <AccountsCards user={this.props.registration.CurrentParent} />
+                </div>
+                {/*<h3>{this.props.registration.CurrentParent.user.name}</h3>*/}
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Registration/SelectParentDialog.js]: Remove this

**Pull request merged**